core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -106,12 +107,19 @@ public KafkaStore(SchemaRegistryConfig config,
     this.zkUtils = ZkUtils.apply(
         kafkaClusterZkUrl, zkSessionTimeoutMs, zkSessionTimeoutMs,
         JaasUtils.isZkSecurityEnabled());
-    this.kafkaTopicReader =
-        new KafkaStoreReaderThread<>(zkUtils, kafkaClusterZkUrl, topic, groupId,
-                                         Integer.MIN_VALUE, this.storeUpdateHandler,
-                                         serializer, this.localStore, this.noopKey);
     this.brokerSeq = zkUtils.getAllBrokersInCluster();
 
+    List<Broker> brokers = JavaConversions.seqAsJavaList(brokerSeq);
```

Comment:
@hachikuji We need to figure out the compatibility story on that. We would prefer that, but we used to be on the old consumer and we need ZK info since schema registry uses it to do it's own leadership election. Since we already have the ZK config, it probably doesn't make sense to provide a separate broker list unless we have a path for getting away from ZK entirely